tools/plasflow/plasflow.xml

```diff
@@ -0,0 +1,39 @@
+<tool id="PlasFlow" name="PlasFlow" version="1.0">
+    <description>Prediction of plasmid sequences in metagenomic contigs</description>
+    <requirements>
+        <requirement type="package" >plasflow</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+PlasFlow.py
+--input '$readFile'
+--output '$output'
```

Comment:
is dit alleen een prefix? dan ik zou hier `--output output` van maken (zonder dollarteken) want `$output` wijst naar iets als `/tmp/f8idhfs/dataset1234.dat` en worden dus alle files daar aangemaakt ipv in de working dir (en werkt 'from_work_dir' ook niet)

Reply:
Het werkt, ongelofelijk!